src/tests/JIT/Regression/JitBlue/Runtime_95315/Runtime_95315.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using Xunit;
+
+public class Runtime_95315
+{
+    [Fact]
+    public static void TestEntryPoint()
+    {
+        for (int i = 0; i < 4; i++)
```

Comment:
Why are the constants in this test chosen to be the values they are?

e.g., is "4" because the constant 4-count loop gets unrolled? 200 because it doesn't get unrolled?

Reply:
It's to make sure we tier up completely (handling up to 4 potential tiers...)